docs/spec/openapi.json

```diff
@@ -9826,39 +9826,6 @@
         "responses": {
           "200": {
             "description": "successful"
-          },
-          "401": {
-            "description": "Unauthorized",
-            "content": {
-              "application/json": {
-                "examples": {
-                  "Unauthorized": {
-                    "value": {
-                      "type": "error.list",
-                      "request_id": "110801d1-2f7b-436f-8f03-2245545a1432",
-                      "errors": [
-                        {
-                          "code": "unauthorized",
-                          "message": "Access Token Invalid"
-                        }
-                      ]
-                    }
-                  }
-                },
-                "schema": {
-                  "$ref": "#/components/schemas/error"
-                }
-              }
-            }
-          }
-        },
-        "requestBody": {
-          "content": {
-            "application/json": {
-              "schema": {
-                "$ref": "#/components/schemas/create_data_event_summaries_request"
```

Comment:
@PasinduDilmin, don't remove `requestBody` details. It is only modifying the `response` section. 
Could you please add the modification for other necessary places ex: I could see `events` endpoints `post` methods,  `tags/{id}` endpoint's `delete` methods have the same behaviour as what we discussed in the previous comment?